### PR TITLE
[backports-release-1.9] add pkgimages to NEWS

### PR DESCRIPTION
**Pull request by IanButterworth**
Once merged here it needs adding to HISTORY on master

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ Language changes
 Compiler/Runtime improvements
 -----------------------------
 
+* The package precompilation process now saves native code into a "pkgimage", meaning that code generated during the
+  precompilation process will not require compilation after package load, resulting in much faster time to first
+  execution (TTFX), often referred to as time to first plot ([#44527]) ([#47184]).
```
Comment (timholy) [inline NEWS.md:30]: Would it be better to start with the consequence, and then describe how it was done? I.e., "Time to first execution (TTFX, sometimes called time to first plot) is greatly reduced. Package precompilation ..."

**Review by timholy — COMMENTED** [review]
Thanks for doing this!

**Comment by giordano** [comment]
Mention also the `--pkgimages` flag?  Also, I presume this should also go into the `HISTORY.md` file, otherwise there will be a mismatch.

File: NEWS.md (hunk 1 above)
Comment (IanButterworth) [inline NEWS.md:30]:
```suggestion
* Time to first execution (TTFX, sometimes called time to first plot) is greatly reduced. Package precompilation now
  saves native code into a "pkgimage", meaning that code generated during the precompilation process will not
  require compilation after package load. Use of pkgimages can be disabled via `--pkgimages=no` ([#44527]) ([#47184]).
```

Comment (IanButterworth) [reply NEWS.md:30]: @timholy look ok? Perhaps we can finalize in a thread to save unnecessary CI runs

**Review by timholy — APPROVED** [review]
LGTM!

File: NEWS.md (hunk 1 above)
Comment (timholy) [reply NEWS.md:30]: Oh, I responded to the PR but I guess I mean I like this version.

Comment (IanButterworth) [reply NEWS.md]: Got it. Thanks

**Comment by JeffBezanson** [comment]
doc/NEWS-update.jl needs to be run.

**Comment by IanButterworth** [comment]
Thanks. Now run on https://github.com/JuliaLang/julia/pull/48311

**Pull request merged**